src/createServer.js

```diff
@@ -1,3 +1,43 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+const http = require('http');
+const { convertToCase } = require('./convertToCase');
+const { getErrorMessage } = require('./getErrorMessage');
+
+function createServer() {
+  const server = http.createServer((req, res) => {
+    res.setHeader('Content-type', 'application/json');
+
+    const [path, query] = req.url.split('?');
+    const textToConvert = path.slice(1);
+    const params = new URLSearchParams(query);
+    const toCase = params.get('toCase');
```

Comment:
Would be good to move logic with reading params from URL to a separate function

Reply:
Fixed.